### PR TITLE
[#138323337]add migration: Add SupplierFramework.prefill_declaration_from_framework_id column and constraints

### PR DESCRIPTION
**Pull request by risicle**
Story https://www.pivotaltracker.com/story/show/138323337

Accompanying model changes will follow with the feature once I've tidied up the commits.

**Edit:** feature PR https://github.com/alphagov/digitalmarketplace-api/pull/526

### CODE REVIEW

**Comment by risicle** [comment]
(see discussion on naming in https://github.com/alphagov/digitalmarketplace-api/pull/526)

**Comment by risicle** [comment]
Oh look it's all magically changed name.

File: migrations/versions/830_add_supplierframework_prefill_declaration.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# revision identifiers, used by Alembic.
+revision = '830'
+down_revision = '820'
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+    op.add_column('supplier_frameworks', sa.Column('prefill_declaration_from_framework_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(op.f('fk_supplier_frameworks_prefill_declaration_from_framework_id_frameworks'), 'supplier_frameworks', 'frameworks', ['prefill_declaration_from_framework_id'], ['id'])
```
Comment (TheDoubleK) [inline migrations/versions/830_add_supplierframework_prefill_declaration.py]: This constraint name seems to be too long for Postgres - it's been truncated to `"fk_supplier_frameworks_prefill_declaration_from_framework_id_fr"` in my DB.,  although the `downgrade` still runs so maybe it's just an issue with psql `\d` display.

Comment (risicle) [inline migrations/versions/830_add_supplierframework_prefill_declaration.py]: what happens if you try and downgrade it?

Comment (TheDoubleK) [inline migrations/versions/830_add_supplierframework_prefill_declaration.py]: Like I said, "the downgrade still runs" and the DB is right, so it should be fine.

**Comment by TheDoubleK** [comment]
:+1:

**Pull request merged**